File: bot.php
<?php
// توکن ربات
$botToken = "7757268669:AAF-oMAe7wuoe6R-2uYrT4Zd3v36P9YX-Dg";
$apiUrl = "https://api.telegram.org/bot$botToken/";

// اطلاعات ایمیل و رمز
$email = "example@email.com";
$password = "example_password";

// دریافت آپدیت از تلگرام
$update = file_get_contents("php://input");
$update = json_decode($update, true);

// بررسی نوع آپدیت
if (isset($update['message'])) {
    $chatId = $update['message']['chat']['id'];
    $firstName = $update['message']['chat']['first_name'] ?? "کاربر";

    if ($update['message']['text'] == "/start") {
        $welcomeText = "سلام $firstName عزیز! 🌟\nخوش اومدی! لطفاً روی دکمه زیر کلیک کن تا ایمیل و رمز برات ارسال بشه.";

        $keyboard = [
            "inline_keyboard" => [
                [["text" => "📩 دریافت ایمیل و رمز", "callback_data" => "get_email_password"]]
            ]
        ];

        $replyMarkup = json_encode($keyboard);

        file_get_contents($apiUrl . "sendMessage?chat_id=$chatId&text=" . urlencode($welcomeText) . "&reply_markup=$replyMarkup");
    }
}

// بررسی کلیک روی دکمه
if (isset($update['callback_query'])) {
    $callbackQuery = $update['callback_query'];
    $chatId = $callbackQuery['message']['chat']['id'];
    $callbackData = $callbackQuery['data'];

    if ($callbackData == "get_email_password") {
        $responseText = "📧 **ایمیل:** $email\n🔑 **رمز عبور:** $password\n\nاز این اطلاعات استفاده کن! 😊";

        file_get_contents($apiUrl . "sendMessage?chat_id=$chatId&text=" . urlencode($responseText) . "&parse_mode=Markdown");
    }
}

File: bot.py
from telegram import Update, InlineKeyboardButton, InlineKeyboardMarkup
from telegram.ext import Updater, CommandHandler, CallbackQueryHandler, CallbackContext

# توکن ربات رو اینجا وارد کن
BOT_TOKEN = 'توکن_اینجا_قرار_بگیره'

# اطلاعات ایمیل و رمز عبور (ثابت برای همه کاربران)
EMAIL = "example@email.com"
PASSWORD = "example_password"

# وقتی کاربر /start رو بزنه
def start(update: Update, context: CallbackContext):
    user = update.message.from_user
    welcome_text = f"سلام {user.first_name} عزیز! 🌟\nخوش اومدی! لطفاً روی دکمه زیر کلیک کن تا ایمیل و رمز برات ارسال بشه."
    
    keyboard = [[InlineKeyboardButton("📩 دریافت ایمیل و رمز", callback_data="get_email_password")]]
    reply_markup = InlineKeyboardMarkup(keyboard)
    
    update.message.reply_text(welcome_text, reply_markup=reply_markup)

# وقتی کاربر روی دکمه کلیک کنه
def button_handler(update: Update, context: CallbackContext):
    query = update.callback_query
    query.answer()
    
    # ارسال ایمیل و رمز عبور
    message = f"""
📧 **ایمیل:** {EMAIL}
🔑 **رمز عبور:** {PASSWORD}

از این اطلاعات استفاده کن! اگه مشکلی داشتی به ادمین پیام بده. 😊
    """
    query.edit_message_text(message)

# راه‌اندازی ربات
def main():
    updater = Updater(BOT_TOKEN, use_context=True)
    dispatcher = updater.dispatcher
    
    dispatcher.add_handler(CommandHandler("start", start))
    dispatcher.add_handler(CallbackQueryHandler(button_handler))
    
    updater.start_polling()
    updater.idle()

if __name__ == '__main__':
    main()
